components/tabbed-nav/src/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+// styled components
+const NavNavContainerli = styled('div')`
+  display: block;
+  padding: 0;
+  margin: 0;
+`;
+
+const NavUl = styled('ul')`
+    list-style: none;
+    margin: 0;
+    border-top: 0;
+    padding: 0;
+    border-bottom: 2px solid #bfc1c3; 
+`;
+
+const Navli = styled('li')`
+  float: left;
```

Comment:
Lets use display: flex for this instead! You can remove all the floats & clears and add this to the `UL`: 
```
display: flex;
align-items: flex-end;
```

Should do it?